assets/base.css

```diff
@@ -2440,3 +2440,30 @@ details-disclosure > details {
   height: 0;
   padding-bottom: var(--ratio-percent);
 }
+
+.content-containers {
+  border-radius: var(--text-boxes-radius);
+  box-shadow: var(--text-boxes-shadow-horizontal-offset)
+    var(--text-boxes-shadow-vertical-offset)
+    var(--text-boxes-shadow-blur-radius)
+    rgba(var(--color-foreground), var(--text-boxes-shadow-opacity));
+  border: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
+}
```

Comment:
`.multicolumn-card::after` is also adding a `box-shadow` which seem to duplicate this one here. Might want to check with @kmeleta just in case.

Reply:
Same for image with text actually. But there, the `box-shadow` seem to be set to none anyway (`.image-with-text:not(.image-with-text--overlap) .grid__item:last-child>*`)